demo-project/conf/base/catalog_04_feature.yml

```diff
@@ -29,6 +29,9 @@
 feature_importance_output: 
   type: pandas.CSVDataSet
   filepath: ${base_location}/04_feature/feature_importance_output.csv
-  layer: feature 
+  metadata:
+    kedro-viz:
+      layer: feature 
+      preview: 2
```

Comment:
I thought we would just do the `preview_args` and not the `preview`?

Reply:
this is just old code. i removed it. thanks for flagging